roles/addons/prometheus_server/tasks/main.yml

```diff
@@ -30,6 +30,23 @@
     - monitoring.exporters.snmp_exporter.with_generator is defined      
     - monitoring.exporters.snmp_exporter.with_generator
 
+- name: "firewalld █ Add services to firewall's {{ prometheus_server_firewall_zone | default('public') }} zone"
+  firewalld:
+    zone: "{{ prometheus_server_firewall_zone | default('public') }}"
+    service: "{{ item }}"
+    immediate: "yes"
+    permanent: "yes"
+    state: enabled
+  when:
+    - ansible_facts.os_family == "RedHat"
+    - ep_firewall | default(false) | bool
+  loop:
+    - 9090/tcp
+    - 9093/tcp
```

Comment:
This should not work: `service: 9093/tcp`.
Is it alertmanager? There is no pre-defined service in EL8 (at least I can't find one). If possible, can you create the service in a previous task? (see [this example gist](https://gist.github.com/btravouillon/d196488211d69d2cbea22c44ae268410))

Reply:
I took thoses ports from prometheus wiki : https://github.com/prometheus/prometheus/wiki/Default-port-allocations